src/assets/assetAttributes.js

```diff
@@ -117,7 +117,7 @@ class AssetAttributes {
    *   .catch((err) => console.log(err));
    */
   create(assetTypeId, assetAttribute = {}) {
-    const requiredFields = ['description', 'label', 'organizationId'];
+    const requiredFields = ['description', 'label'];
```

Comment:
So I took a look at the creation of Asset Type and Asset Metric to see how we were handling `null` there. We still require `organizationId` as a `requiredField` but we do it a little differently. See [Asset Metric](https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/assets/assetMetrics.js#L102-L122) for reference and [Asset Type](https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/assets/assetTypes.js#L68-L83) as well. 

Also there's an added test for creating with a global scope [here](https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/assets/assetTypes.spec.js#L122) which should probably be added to this as well.